dama/doctrine-test-bundle/4.0/config/packages/dmaicher_doctrine_test_bundle.yaml

```diff
@@ -0,0 +1,4 @@
+dama_doctrine_test:
```

Comment:
This should be moved to `packages/test/` as the bundle is only available in the test environment.